include/api/wasmedge/wasmedge.h

```diff
@@ -48,7 +48,7 @@ typedef struct WasmEdge_Value {
   // functions. Developers should use the corresponding `WasmEdge_ValueGen`
   // functions to generate this struct, and the `WasmEdge_ValueGet` functions to
   // retrieve the value from this struct.
-  enum WasmEdge_ValType Type;
+  WasmEdge_FullValType Type;
```

Comment:
This is a C-style header. Should we keep the `enum` keyword here?

Reply:
Another thing is that you changed the enum/class name without modifying the corresponding comments.

Reply:
The `WasmEdge_FullValType` is a struct rather than a `enum`. The `struct` keyword before the type name is eliminated by the `typedef` in the code that defines the struct, so I think there is no need to add a keyword before it.

Reply:
Ah, I misunderstood this line.